pkg/preparer/setup.go

```diff
@@ -116,9 +94,9 @@ func New(preparerConfig *PreparerConfig, logger logging.Logger) (*Preparer, erro
 	}
 
 	var err error
-	var keyring openpgp.KeyRing = nil
+	var verifier auth.SigVerifier
 	if preparerConfig.KeyringPath != "" {
-		keyring, err = loadKeyring(preparerConfig.KeyringPath)
+		verifier, err = auth.FileKeyringVerifier(preparerConfig.KeyringPath)
```

Comment:
I don't know if you guys will like this idea, but I sort of think that we should get rid of the empty KeyringPath check above, and change `auth.FileKeyringVerifier` to expect an empty string and give you a verifier that always returns true. That gets us away from having to check if auth.SigVerifier is a zero value, etc